server/controllers/getCollection.js

```diff
@@ -9,7 +10,9 @@ export default async function createGetCollectionHandler(req, res) {
         const requestHeaders = {
             [headers.BggFilterPlayerCount]: req.query.numPlayers,
             [headers.BggFilterMinDuration]: req.query.minDuration,
-            [headers.BggFilterMaxDuration]: req.query.maxDuration
+            [headers.BggFilterMaxDuration]: req.query.maxDuration,
+            [headers.BggFieldWhitelist]: 'name,maxplayers,minplayers,maxplaytime,image,thumbnail'

```

Comment:
This can also be deduplicated.  If we have the whitelist in one place it'll be easier to adjust in the future.